### DISCUSSION TITLE
Update problem

### DISCUSSION THREAD

**Discussion by Meowkun09** [Monika-After-Story/MonikaModDev#7936] [QA - tech support + general] [answered]
Im trying to update the new version but it wont let me. Anybody help please.
![Screenshot 2021-09-22 203223](https://user-images.githubusercontent.com/91198126/134344080-dbc51494-5655-4668-bfc3-02d0c7b4ad59.png)

**Comment by ghost** [comment]
> Im trying to update the new version but it wont let me. Anybody help please.
> ![Screenshot 2021-09-22 203223](https://user-images.githubusercontent.com/91198126/134344080-dbc51494-5655-4668-bfc3-02d0c7b4ad59.png)

i wanna try and help, can you send me the files of the game so i can see whats wrong?

**Comment by Meowkun09** [comment]
> > Im trying to update the new version but it wont let me. Anybody help please.
> > ![Screenshot 2021-09-22 203223](https://user-images.githubusercontent.com/91198126/134344080-dbc51494-5655-4668-bfc3-02d0c7b4ad59.png)
> 
> i wanna try and help, can you send me the files of the game so i can see whats wrong?

whole file?

**Comment by ghost** [comment]
> > > Im trying to update the new version but it wont let me. Anybody help please.
> > > ![Screenshot 2021-09-22 203223](https://user-images.githubusercontent.com/91198126/134344080-dbc51494-5655-4668-bfc3-02d0c7b4ad59.png)
> > 
> > 
> > i wanna try and help, can you send me the files of the game so i can see whats wrong?
> 
> whole file?

mhm! :D im not a dev or anything, but i wanna try to help people!

**Comment by ghost** [comment]
just send the zip or something of the game.

**Comment by Meowkun09** [comment]
> just send the zip or something of the game.

uhh it wont let me...
![Screenshot 2021-09-22 211030](https://user-images.githubusercontent.com/91198126/134349741-58ca9ff3-0975-4deb-9e04-fffadb3bfe4b.png)

**Comment by ThePotatoGuy** [comment, marked as answer]
try manually updating MAS by reinstalling - see step 3 of [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-still-cant-run-the-game-and-none-of-the-above-faqs-solve-my-issue)

**Reply by Meowkun09** [reply]
> try manually updating MAS by reinstalling - see step 3 of [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-still-cant-run-the-game-and-none-of-the-above-faqs-solve-my-issue)

Reinstalling? Will i lose my data?

**Reply by ThePotatoGuy** [reply]
no
